### DISCUSSION TITLE
欢迎来到 LEDE 讨论区

### DISCUSSION THREAD

**Discussion by coolsnowwolf** [coolsnowwolf/lede#9508] [Announcements / 公告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smeor** [comment]
大雕威武

**Comment by kino516** [comment]
大雕威武

**Comment by smeor** [comment]
## 有谁不服

**Comment by tian88a** [comment]
人生的第一台软路由，再过两天就要到了，求出一份r68s详细新手教程。。感谢大雕~~~

**Comment by blue-troy** [comment]
讨论区功能是不错

**Comment by lbhzy** [comment]
我为大雕举大旗👻

**Comment by tanbingqin** [comment]
/home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0/gcc/config/aarch64/aarch64-bti-insert.c:194: undefined reference to `gen_bti_j()'
/usr/bin/ld: aarch64-bti-insert.o: in function `rest_of_insert_bti':
/home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0/gcc/config/aarch64/aarch64-bti-insert.c:194: undefined reference to `gen_bti_j()'
/usr/bin/ld: /home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0/gcc/config/aarch64/aarch64-bti-insert.c:183: undefined reference to `gen_bti_j()'
/usr/bin/ld: /home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0/gcc/config/aarch64/aarch64-bti-insert.c:212: undefined reference to `gen_bti_c()'
collect2: error: ld returned 1 exit status
make[5]: *** [/home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0/gcc/c/Make-lang.in:87: cc1] Error 1
make[5]: Leaving directory '/home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0-initial/gcc'
make[4]: *** [Makefile:4428: all-gcc] Error 2
make[4]: Leaving directory '/home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0-initial'
make[3]: *** [Makefile:37: /home/jong/lede/build_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/gcc-11.3.0-initial/.built] Error 2
make[3]: Leaving directory '/home/jong/lede/toolchain/gcc/initial'
time: toolchain/gcc/initial/compile#5.82#4.95#10.67
    ERROR: toolchain/gcc/initial failed to build.
make[2]: *** [toolchain/Makefile:97: toolchain/gcc/initial/compile] Error 1
make[2]: Leaving directory '/home/jong/lede'
make[1]: *** [toolchain/Makefile:93: /home/jong/lede/staging_dir/toolchain-aarch64_generic_gcc-11.3.0_musl/stamp/.toolchain_compile] Error 2
make[1]: Leaving directory '/home/jong/lede'
make: *** [/home/jong/lede/include/toplevel.mk:231：world] 错误 2
编译到这里报错怎么解决啊？
